CHANGELOG.md

```diff
@@ -0,0 +1,139 @@
+# Release 2.0
+
+## What's new
+
+This is a brief development summary for this release.
+
+For a list of relevant Pull Requests related to the 2.0 release, see this [GitHub query result][].
+
+### Server platform and language independence
+
+-   UPWARD spec introduced for cross platform server behavior configuration.
+    An UPWARD file describes a stack-agnostic server that is used for server side rendering, asset delivery, and proxying.
+-   A PWA package can now define its network requirements in an UPWARD file.
+-   A PWA can now be deployed on any tech stack as long as the server is UPWARD compliant.
+-   UPWARD servers implemented in[NodeJS][] and [PHP][]
+-   UPWARD spec published and open for community contribution.
+
+#### Related documentation updates
+
+-   [UPWARD spec overview][]([#382][])
+-   [UPWARD-js server][]([#382][])
+
+### Improved developer experience
+
+-   Implemented consistent patterns for workflow, such as action/reducer organization and container/presentational separation, in the Venia reference theme source code.
```

Comment:
Everywhere else, Venia is referred to as a "reference storefront" as opposed to a "reference theme". Since "theme" means something very specific to Magento developers, I think we should change this.

Reply:
Updated